docs/introduction.rst

```diff
@@ -79,12 +79,12 @@ Supported environments
 
 .. _pywbem documentation: https://pywbem.readthedocs.io/en/stable/intro.html#wbem-servers
 
-The pywbemcli module of the pywbemtools package is supported in these
-environments:
+The pywbemtools package is supported in these environments:
 
-* Operating systems: Linux, Windows, macOS
+* Operating systems: Linux, Windows (native and Unix-like environments
+  (ex. Cygwin), OS-X
 * Python versions: 2.7, 3.4, and greater
-* pywbem 0.14.4 or greater
+* pywbem
```

Comment:
We can remove the "pywbem" item entirely. If we don't specify the minimum version here, then there is nothing on it that is worth mentioning in a section about supported environments.

Reply:
Agreed. Removed